unic/utils/src/lib.rs

```diff
@@ -9,7 +9,7 @@
 // except according to those terms.
 
 
-#![forbid(unsafe_code, missing_docs)]
+#![forbid(missing_docs)]
```

Comment:
Let's make unsafe usage more explicit. Here's how we can do ti: add
```
#![deny(unsafe)]
```
here, then on the module with unsafe need, add
```
#![allow(unsafe)]
```